.github/workflows/test_e2e.yml

```diff
@@ -12,6 +12,8 @@ on:
       - master
     types:
       - completed
+  # Allow manual runs
+  workflow_dispatch:
```

Comment:
hmm, i think may need the inputs field here, if we're pasting a url in? :O 

or currently its more to trigger it when needed~ xD

Reply:
trigger when needed, so no url needed, check out the code here: https://github.com/sebastianquek/github-actions-test/blob/master/.github/workflows/test_env_var.yml

```
on:
  push:
  pull_request:
  workflow_dispatch:
```

Reply:
okk! got it haha~ thanks~